### DISCUSSION TITLE
Add ability to force recreate specific container when compose up

### DISCUSSION THREAD

**Discussion by sky-stroller** [louislam/dockge#291] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

When compose up, force recreate container for some services.
I'm using unraid, and some sercvices (qbittorrent eg.) use external network 'br0' which created by unraid. Unraid recreate 'br0' when docker service restart. And then services using 'br0' cannot be stared because of id of 'br0' has changed. I have to down the stack and up again manually.

### ✔️ Solution

* add a checkbox to mark which container should be force recreated when compose up. 
or 
* add text field to specify extra options for 'docker compose up'

**Comment by geozeke** [comment]
^ This please! 🙏 🙂

**Comment by snok3r-krk** [comment]
+1
